public/catalog/v1.0.0/openapi.json

```diff
@@ -1340,7 +1340,8 @@
               "Approval Pending",
               "Ordered",
               "Failed",
-              "Completed"
+              "Completed",
+              "Approved"
```

Comment:
@lindgrenj6 Is this for the order or order_item? We have to keep in mind that an order can have multiple order_items, the order_items individually will have the denied/approved etc but not the order. The Order goes to Ordered -> Completed or Ordered -> Failed. All the extra details would be in the individual order items.

Reply:
It was for order, I pulled out the approved state for order.